src/store/modules/endpoints.js

```diff
@@ -96,16 +96,20 @@ export const useEndpointStore = defineStore('endpoint', {
 				: `/index.php/apps/openconnector/api/endpoints/${endpointItem.id}`
 			const method = isNewEndpoint ? 'POST' : 'PUT'
 
+			// Create a copy of the endpoint item and remove empty properties
+			const endpointToSave = { ...endpointItem }
```

Comment:
Dit maakt een shallow copy wat niet de bedoeling is, gebruik eerder de cloneDeep (`_.cloneDeep(value)`) functie van Lodash